Makefile

```diff
@@ -187,7 +190,12 @@ uninstall: manifests kustomize kubectl ## Uninstall CRDs from the K8s cluster sp
 deploy: manifests kustomize kubectl ## Deploy controller to the K8s cluster specified in ~/.kube/config.
 	@$(KUBECTL) create configmap env-config --from-literal=HWMGR_PLUGIN_NAMESPACE=$(HWMGR_PLUGIN_NAMESPACE) --dry-run=client -o yaml > config/manager/env-config.yaml
 	cd config/manager && $(KUSTOMIZE) edit set image controller=${IMG}
+	cp config/manager/inventory.yaml config/manager/inventory.back
+	sed -i 's/ingressHost:.*/ingressHost: $(INGRESS_HOST)/' config/manager/inventory.yaml
+	cat config/manager/inventory.yaml
 	$(KUSTOMIZE) build config/$(KUSTOMIZE_OVERLAY) | $(KUBECTL) apply -f -
+	cp config/manager/inventory.back  config/manager/inventory.yaml
+	rm -f config/manager/inventory.back
```

Comment:
I'm not really sure I understand what is being accomplished with the copying of the inventory.yaml file.   Could you explain why this is necessary.